### PR TITLE
docs: readme: add new ppa

### PR DESCRIPTION
**Pull request by muzena**
@Lekensteyn 
I made a ppa repository which when you make a change in github automatically syncs and builds debs packages.

### CODE REVIEW

**Comment by Lekensteyn** [comment]
Hi @muzena, thank you for your packaging attempt. However, as part of a migration to a new repo, we will also rework the packaging and create a new PPA in due time, so I will close this for now.

**Comment by bdcomp** [comment]
This issue should be reopened as the readme still not been updated.

**Comment by FFY00** [comment]
@muzena can you rebase?

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Pre-built packages are available for a few Linux distros.
 * Debian 7 (Wheezy) or higher: packages in this [repository](docs/debian.md)
 * Ubuntu/Kubuntu 16.04+: use the `solaar-gnome3` and/or `solaar` package from [universe repository][universe repository]
 * Ubuntu/Kubuntu 12.04 to 14.04: use `solaar` package from [ppa:daniel.pavel/solaar][ppa]
+* Ubuntu/Kubuntu 12.04 to 19.04 git build packages: use `solaar-gnome3` and/or `solaar`  package from [ppa:solaar-unifying/ppa][ppa1]
```
Comment (bdcomp) [inline README.md]: 19.04 -> 20.04

File: README.md [hunk 2]
```diff
@@ -35,6 +36,7 @@ integration with *gnome-shell* or *Unity*, install `solaar-gnome3`.
 * a [Mageia package][mageia], courtesy of David Geiger
 * an [OpenSUSE rpm][opensuse], courtesy of Mathias Homann
 
+[ppa1]: https://launchpad.net/~solaar-unifying/+archive/ubuntu/ppa
```
Comment (bdcomp) [inline README.md]: I think that a link to the Stable PPA is better here https://launchpad.net/~solaar-unifying/+archive/ubuntu/stable

File: README.md (hunk 1 above)
Comment (FFY00) [reply README.md]: Ah, yes. Updating this should also be good.

**Comment by muzena** [comment]
@bdcomp 
I made a stable PPA repository after this pull request.
Can you give me a some guide or example how to rebase this pull request?

**Comment by FFY00** [comment]
`git fetch origin` and `git rebase origin/master` (assuming that you originally cloned this repo)

If you did not originally cloned this repo then you need to add a new remote `git remote add pwr-solaar https://github.com/pwr-Solaar/Solaar.git` and replace `origin` with `pwr-solaar`. You can use a name other than `pwr-solaar`.

**Comment by FFY00** [comment]
This commit is replacing the redirect that we have from `README.md` to `docs/index.md`, can you please not do that?

**Comment by muzena** [comment]
@FFY00 
Is now ok?

File: README.md [hunk 3]
```diff
@@ -1 +1 @@
-docs/index.md
\ No newline at end of file
+./docs/index.md
```
Comment (FFY00) [inline README.md]: This is not needed but I can overlook if you don't have time to fix it.

Comment (muzena) [reply README.md]: Problem is in symlink?

Comment (FFY00) [reply README.md]: No, the addition of `./`. I am just nitpicking here.

**Comment by muzena** [comment]
@FFY00 
I think that is now finally ok?

**Comment by FFY00** [comment]
Perfect! Thanks for bearing with me!

**Pull request merged**